csharp/src/Ice/InputStream-Class.cs

```diff
@@ -91,10 +92,12 @@ public RemoteException ReadException()
 
                     ReadIndirectionTableIntoCurrent(); // we read the indirection table immediately.
 
-                    if (Communicator.FindRemoteExceptionFactory(typeId) is Func<string?, RemoteException> factory)
+                    Func<string?, RemoteExceptionOrigin?, RemoteException>? factory =
```

Comment:
Why not use the  `if (... is Func<> factory)` like before?

Reply:
just to split it into two lines, was too long with the new generic parameter

Reply:
reverted to previous style